README.md

```diff
@@ -753,7 +753,11 @@ public class MyComponentDialog { /* ... */ }
 
 #### Fields inheritance and ways to cancel it
 
-Same as dialog tabs, dialog fields are "inherited" across the Java classes. However, there is the possibility to "cancel" a superclass-bound field from rendering in a current dialog or a narrower container. Add `@IgnoreFields` annotation to the current class:
+Same as dialog tabs, dialog fields are "inherited" across Java classes. Dialog fields from a superclass and from a child class are always considered two different fields, event if sharing same name. If a superclass contains a field like `@DialogField @TextField private String text;`, and its child class contains a `@DialogField @TextField private String text;` of its own, *both*  will be rendered in a dialog for the child class.
```

Comment:
event if -> even if

Reply:
> If a superclass contains a field like `@DialogField @TextField private String text;`, and its child class contains a `@DialogField @TextField private String text;` of its own, *both*  will be rendered in a dialog for the child class.

not exactly true. only the latter field will be rendered if they are both under the same node, but it will be placed in the position of the fist field. Could you please reflect that in the docs?